modules/ROOT/pages/map-users-roles-ldap-task.adoc

```diff
@@ -1,39 +1,152 @@
-= To Map Users to Roles in an LDAP Group
+= Map Single Sign-On Users to Roles
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-You can map users in a federated organization's LDAP group to an Anypoint role. Your Anypoint Platform organization needs to use an external identity provider, such as PingFederate.
+You can map users in a federated organization's group to an Anypoint role. Your Anypoint Platform organization must use an external identity provider, such as PingFederate.
 
-After you perform this mapping, users in an organization can sign into Anypoint Platform using the same organizational credentials and access permissions that an organization maintains using LDAP. +
+After you perform this mapping, users in an organization can sign in to Anypoint Platform using the same organizational credentials and access permissions that an organization maintains using SAML, OpenID Connect (OIDC), or LDAP. +
 This ensures security of credentials and maintains organizational roles for accessing privileged information.
 
-Choose an external identity provider and then configure roles.
+Anypoint Platform requires different information from your identity provider based on whether you use SAML, OpenID Connect, or LDAP.
 
-To configure a role:
+== Configure Multiple Group Mappings for Single Sign-On using SAML
+
+If your organization uses single sign-on and requires its users to obtain permissions from multiple groups, you can configure a group attribute that contains all the groups you want to use. In many cases, you can create an array of groups.
```

Comment:
I got lost between "requires its users" and "you want to use," so how about...:
```suggestion
If your organization uses single sign-on but also requires you to obtain permissions from multiple groups, you can configure an attribute that contains all of the groups from which you need to obtain individual access permission. In many cases, you can create an array of groups.
```

Reply:
What is "an array of groups" vs. "an attribute" that contains the groups?

Reply:
Array in this case is used in the context of coding/scripting.